src/sql/platform/connection/common/connectionManagement.ts

```diff
@@ -71,6 +71,7 @@ export interface IConnectionResult {
 	callStack: string;
 	errorHandled?: boolean;
 	connectionProfile?: IConnectionProfile;
+	uriForPasswordChange?: string;
```

Comment:
should this be name something more generic such as `connectionUri` or is this a different URI?